commands/upgrade/upgrade_path.py

```diff
@@ -0,0 +1,124 @@
+import json
+import os
+
+from leapp.exceptions import CommandError
+from leapp.utils import path
+
+
+HANA_BASE_PATH = '/hana/shared'
+HANA_SAPCONTROL_PATH = 'exe/linuxx86_64/hdb/sapcontrol'
+
+LEAPP_UPGRADE_FLAVOUR_DEFAULT = 'default'
+LEAPP_UPGRADE_FLAVOUR_SAP_HANA = 'saphana'
+# XXX FIXME NOTE(ivasilev) Need to agree on a hardcoded path to store the file (and package it properly)
+# or implement config file for leapp-repository.
+# We can't use api.get_common_file_path functions here as there is no actor context
+# XXX FIXME Hardcoding just for the sake of testing to deal with other issues
+LEAPP_UPGRADE_PATHS = 'upgrade_paths.json'
+
+
+# XXX FIXME NOTE(ivasilev) Shamelessly copied from leapp.libraries.common.config as could not include shared actor
+# library at the command level
```

Comment:
I think that this functino is so generic the comment is not needed.

Reply:
I removed XXX FIXME but left the note - imho it's good to know that actor library can't be loaded at the command level.